pkg/repositories/transformers/execution.go

```diff
@@ -99,6 +99,11 @@ func CreateExecutionModel(input CreateExecutionModelInput) (*models.Execution, e
 	}
 
 	activeExecution := int32(admin.ExecutionState_EXECUTION_ACTIVE)
+	tags := make([]models.AdminTag, len(input.RequestSpec.Tags))
+	for i, tag := range input.RequestSpec.Tags {
+		tags[i] = models.AdminTag{Name: tag}
+	}
```

Comment:
This will end up creating separate rows in the tags table that contain the same value, right? As in, if I create two executions in which both set the tag "tagA", then we'll end up creating two rows in the Tags table with the name "tagA", correct?

Reply:
I just set the `tag_name` column as unique so we won't create duplicate tags.

Reply:
do we have to worry about fetching those tags? What happens if we try to save an execution that contains a new tag that's a duplicate? Does this cause an error?

Reply:
No, it doesn't cause an error, and won't create a new row having the same tag.

Reply:
The integration test shows a few errors: https://github.com/flyteorg/flyteadmin/actions/runs/5717905981/job/15492760870?pr=571#step:7:346.